docs/release.md

```diff
@@ -0,0 +1,8 @@
+# Release process for composio core and plugin packages
```

Comment:
This can be automated down the line once we have a streamlined release process, for now it'll be manual